### ISSUE TITLE
AoA statement about representing GR

### ISSUE THREAD

**Issue by 777arc** [gnuradio/gr-governance#12]
Clearly the wording in the AoA about the board representing GR is not adequate, it's a shame folks didn't bring this up before we voted in the AoA but whatever, we'll deal with it now via whatever voting process required. Here's what it says now:

5. Each individual board member has the right to represent the Association to outside parties.

And here's my proposal, feel free to provide suggestions.  

5. Each individual board member has the right to represent the Association to outside parties regarding legal and contractual purposes.  This does not preclude the other GA members from representing the project in non-legal and non-contractual situations, as long as it's done in a reasonable manner, especially when related to their roles within the project (e.g. lead code maintainer representing the project for matters related to the code).

**Comment by mbr0wn** [comment]
@777arc your first suggestion seems a bit wordy to me, and I'm not sure what implications the "legal representation" has. It sounds like any GA member could sign contracts on whatever, and that could be risky. I very much like your second suggestion.

**Comment by 777arc** [comment]
Yep that's fair

**Comment by noc0lour** [comment]
> I definitely don't want the list of roles to be in the AoA. They should definitely be listed on the website, and they will be in our meeting minutes.

The list of currently elected/appointed folks should also be tracked in a simple list in this repository, e.g. 
```
# $role
- Appointee 1 (elected on dd.mm.yyyy)
- Appointee 2 (elected on dd.mm.yyyy)
# $role2
- Appointee 3 (elected on dd.mm.yyyy)
# $role3
- Appointee 1 (elected on dd.mm.yyyy)
```

structure depends on the agreed upon procedure in the AoA of course (w.r.t to multiple memberships, teams etc).

**Comment by duggabe** [comment]
In my perspective, I recommend:

* the term "represent" needs to be defined in the Clarification of Terminology section
* Use Marc's second wording "The GA appoints individuals as Leads for specific areas of responsibilities which the Leads then have primary decision making powers over. The GA appoints a Lead by simple majority voting and can replace a Lead by another vote."
* Use noc0lour's suggestion
> The list of currently elected/appointed folks should also be tracked in a simple list in this repository, e.g.
> 
> ```
> # $role
> - Appointee 1 (elected on dd.mm.yyyy)
> - Appointee 2 (elected on dd.mm.yyyy)
> # $role2
> - Appointee 3 (elected on dd.mm.yyyy)
> # $role3
> - Appointee 1 (elected on dd.mm.yyyy)
> ```
> 
> structure depends on the agreed upon procedure in the AoA of course (w.r.t to multiple memberships, teams etc).

* If we have consensus on this, I think we should proceed to formal adoption.

**Comment by 777arc** [comment]
Yeah I like that, although we sort of pivoted from the original name of the issue, which was just fixing that one line in the AoA, and now we're designing the new technical roles thing.  So we should probably close this out and make a new proposal with less clutter.

**Comment by duggabe** [comment]
What is the formality for making an item for a vote? Creating that issue / agenda / whatever would be the most productive next step.

**Comment by 777arc** [comment]
Agreed
